generated/user_existence/user_existence.json

```diff
@@ -0,0 +1,29 @@
+{
+  "primary_applicant": {
+    "id": "58bf3bb6e41b0738bf0000ce",
+    "user_id": "58bf3b87e41b0738bf0000cc",
+    "first_name": "Loren",
+    "last_name": "Dimallo"
```

Comment:
Yep. I assumed there was going to be some layer of security on this API but I guess not? In any case, recall that we spent all of 5 minutes discussing this API, so I presumed the plan was to have a working version (_which it is, right now_) and then, tweak it.